src/main/java/teammates/ui/controller/InstructorCourseEnrollPageData.java

```diff
@@ -1,19 +1,30 @@
 package teammates.ui.controller;
 
 import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.util.Sanitizer;
 
 /**
  * PageData: this is page data for 'Enroll' page for a course of an instructor
  */
 public class InstructorCourseEnrollPageData extends PageData {
+    private String courseId;
+    private String enrollStudents;
     
-    public InstructorCourseEnrollPageData(AccountAttributes account) {
+    public InstructorCourseEnrollPageData(AccountAttributes account, String courseId, String enrollStudents) {
         super(account);
-        enrollStudents = "";
+        this.courseId =  Sanitizer.sanitizeForHtml(courseId);
+        this.enrollStudents = Sanitizer.sanitizeForHtml(enrollStudents);
```

Comment:
I believe `sanitizeForHtml()` is availabe from `PageData`, so there's no need for the static reference to Sanitizer.